pxc/local/build-binary-galera3

```diff
@@ -55,6 +55,10 @@ fi
 # ------------------------------------------------------------------------------
 
 pushd ${SOURCEDIR}
-    scons
+    if [[ "${CMAKE_BUILD_TYPE}" = "Debug" ]]; then
+       scons debug=0 --config=force
```

Comment:
`--config=force` may not work properly if we are building unittests. Please check this once locally.
```
/usr/bin/ld: /home/venki/work/pxc/3353/percona-xtradb-cluster-galera/galerautils/tests/gu_utils_test.c:64: undefined reference to `_mark_point'
/usr/bin/ld: /home/venki/work/pxc/3353/percona-xtradb-cluster-galera/galerautils/tests/gu_utils_test.c:65: undefined reference to `_ck_assert_failed'
/usr/bin/ld: /home/venki/work/pxc/3353/percona-xtradb-cluster-galera/galerautils/tests/gu_utils_test.c:65: undefined reference to `_mark_point'
/usr/bin/ld: /home/venki/work/pxc/3353/percona-xtradb-cluster-galera/galerautils/tests/gu_utils_test.c:68: undefined reference to `_ck_assert_failed'
/usr/bin/ld: /home/venki/work/pxc/3353/percona-xtradb-cluster-galera/galerautils/tests/gu_utils_test.c:68: undefined reference to `_mark_point'
/usr/bin/ld: /home/venki/work/pxc/3353/percona-xtradb-cluster-galera/galerautils/tests/gu_utils_test.c:69: undefined reference to `_ck_assert_failed'
```

I guess it is probably better to just do
```
pushd ${SOURCEDIR}
    if [[ "${CMAKE_BUILD_TYPE}" = "Debug" ]]; then
       scons debug=0 --config=force libgalera_smm.so garb/garbd
    else
       scons --config=force libgalera_smm.so garb/garbd
    fi
popd
```